### DISCUSSION TITLE
Bug in symfony blog Categories

### DISCUSSION THREAD

**Discussion by freiondrej-lmc** [symfony/symfony#45934] [Q&A] [answered]
Hello,

when clicking category names in the "Published in" section of the blog post row on https://symfony.com/blog/, it leads to a 404 page. There seems to be missing lowercase and replacing spaces with dashes. I found no repo to create a fixing PR in, so I'd like to ask here, where to report this problem?

**Comment by stof** [comment]
@javiereguiluz can you have a look at that, as that's related to the website ?

**Comment by javiereguiluz** [comment, marked as answer]
@freiondrej-lmc thanks for reporting this error. We've just deployed a fix for it, so it now works as expected. Cheers!
